libats-messaging/src/main/scala/com/advancedtelematic/libats/messaging/ListenerMonitor.scala

```diff
@@ -0,0 +1,66 @@
+package com.advancedtelematic.libats.messaging
+
+import akka.http.scaladsl.util.FastFuture
+import com.advancedtelematic.libats.http.HealthMetrics
+import com.codahale.metrics.{Metric, MetricFilter, MetricRegistry}
+import io.circe.Json
+import org.slf4j.LoggerFactory
+import io.circe.syntax._
+
+import scala.concurrent.Future
+
+trait ListenerMonitor {
+  def onProcessed: Future[Unit]
```

Comment:
`record-lag` metric kafka provides is probably more interesting. We already can expose them, and Auth+ even shows it on its dashboard - http://grafana.gw.prod01.internal.advancedtelematic.com/dashboard/db/auth.